### PR TITLE
#15370 patch 2

### PR DESCRIPTION
**Pull request by GaurangTandon**
#### References to other Issues or PRs

#15370

#### Brief description of what is fixed or changed

More docstrings added. 

#### Other comments

I intend to add more documentation in patches (that way it is also easy to review and maintain). So, this is the second patch after my previous PR.

#### Release Notes

* calculus 
  * Improve docstring for `function_range`, `not_empty_in`, `_periodicity`, and `periodicity` in `util.py`

### CODE REVIEW

**Comment by GaurangTandon** [comment]
Could someone please help me understand why the tests in `sympy/simplify/tests/test_hyperexpand.py:test_partial_simp`, when all I've done is edit documentation comments (not edited code) and all my edits are restricted to `/sympy/calculus/utils.py`?

**Comment by jksuom** [comment]
It seems to fail in a test running hyperexpand on a function with random parameters (https://github.com/sympy/sympy/blob/master/sympy/simplify/tests/test_hyperexpand.py#L637). I'll restart the test (with new parameters).

File: sympy/calculus/util.py [hunk 1]
```diff
@@ -539,6 +581,16 @@ def lcim(numbers):
     The numbers can be rational or irrational or a mixture of both.
     `None` is returned for incommensurable numbers.
 
+    Parameters
+    ==========
+    numbers : int[]
+        Numbers whose lcim has to be founed. They can be rational or irrational or a mixture of both.
```
Comment (smichr) [inline sympy/calculus/util.py]: founed -> found

What does `int[]` mean?

Comment (GaurangTandon) [reply sympy/calculus/util.py]: Sorry, I was just trying to denote an integer array. What other way is to do that?

Comment (smichr) [reply sympy/calculus/util.py]: If the docstring is correct and "they can be rational or irrational or a mixture of both" then this isn't an integer array.
```
numbers: list
    Numbers (rational and/or irrational) for which lcim is to be found.
```

Comment (GaurangTandon) [reply sympy/calculus/util.py]: Hi, thanks, I have fixed it.

**Comment by GaurangTandon** [comment]
Hi, I have made the necessary changes.

**Comment by oscargus** [comment]
Thanks!

**Pull request merged**